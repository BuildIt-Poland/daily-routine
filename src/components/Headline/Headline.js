import styled from 'styled-components';

import media from '../../styles/media';
import { spacingSmall, fontXLarge } from '../../styles/designTokens';

const Headline = styled.h1`
  font-size: ${fontXLarge};
  margin: 0;
  margin-bottom: ${spacingSmall};
  text-align: center;

  ${media.phone`
    text-align: left;
  `};
`;

export default Headline;
