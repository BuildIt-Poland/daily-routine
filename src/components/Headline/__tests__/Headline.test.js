import React from 'react';
import { create } from 'react-test-renderer';

import Headline from '../Headline';

describe('COMPONENT - Headline', () => {
  it('renders correctly', () => {
    const component = create(<Headline />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
