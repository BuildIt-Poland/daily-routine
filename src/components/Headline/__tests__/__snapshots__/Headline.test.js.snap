// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Headline renders correctly 1`] = `
.c0 {
  font-size: 2.4rem;
  margin: 0;
  margin-bottom: 1rem;
  text-align: center;
}

@media (max-width:26.25em) {
  .c0 {
    text-align: left;
  }
}

<h1
  className="c0"
/>
`;
